chromadb/test/property/test_collections_with_database_tenant_overwrite.py

```diff
@@ -1,214 +0,0 @@
-from typing import Dict, Optional, Tuple
```

Comment:
i think we need some flavor of a test of this, it tests important client side functionality